rootfs/etc/polkit-1/rules.d/41-steamvr.rules

```diff
@@ -0,0 +1,10 @@
+/* Allow members of the wheel group to execute the defined actions 
```

Comment:
doesn't this allow any action?

Reply:
No, this should only allow users in the `wheel` group to execute the "org.gameros.steamvr.pkexec" action, which narrowly defines running setcap on vrcompositor-launcher.

Reply:
oh, yeah, my mistake.